src/navigation/MainTabNavigator.tsx

```diff
@@ -6,7 +6,7 @@ import { createStackNavigator, createBottomTabNavigator } from 'react-navigation
 import TabBarIcon from '../shared_components/TabBarIcon'
 import HomeScreen from '../screens/Home/HomeScreen/HomeScreen'
 import OnboardingScreen from '../screens/Home/OnboardingScreen/OnboardingScreen'
-import CreateScreen from '../screens/Home/CreateScreen/CreateScreen'
+import CreateScreen from '../screens/Home/RenderingVideoScreen/RenderingVideoScreen'
```

Comment:
can you import it as RenderingVideoScreen here and then update it on line 36. This screen will now live in the "Create" stack, which is just all the screens that are in the 2nd tab of our app, so it will grow over time. For the merge conflict, I will fix it after you push these changes!